File: backend/customer/src/main/java/com/gmech/customer/customer/Customer.java
package com.gmech.customer.customer;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Entity
public class Customer {
    @Id
    @GeneratedValue
    private Integer id;

    private String name;
    private String country;
    private String postCode;
    private String street;
    private String houseNumber;
    private String email;
    private String phoneNumber;
    private String taxNumber; 
}

File: backendLaravel/app/Models/customer.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class customer extends Model
{
    use HasFactory;
    protected $fillable = [
         'name', 'country', 'postcode', 'street', 'houseNumber','email' , 'phoneNumber', 'taxNumber'
        ];

    public function quotation()
    {
        return $this->hasMany(Quotation::class, 'customerId', 'id',);
    }
}
